### PR TITLE
update release script and release process

### PR DESCRIPTION
**Pull request by jessfraz**
It is time, ping @tianon @cpuguy83

### CODE REVIEW

**Comment by calavera** [comment]
LGTM

**Comment by cpuguy83** [comment]
:+1: I know nothing about the release process, so those changes ought to be looked at by someone who's actually done it/looked at it in some way :)

**Comment by calavera** [comment]
@jfrazelle this doesn't look right:

```
hack/make/ubuntu: line 139: fpm: command not found
```

It's at the end of the janky build.

**Comment by jfrazelle** [comment]
oooo i need to remove in bundles!

**Comment by jfrazelle** [comment]
yay more minus!

**Comment by jfrazelle** [comment]
updated!

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-1.10.0-dev
+1.6.1
```
Comment (tianon) [inline VERSION]: uhhhh

File: contrib/builder/deb/debian-jessie/Dockerfile [hunk 2]
```diff
@@ -6,7 +6,7 @@ FROM debian:jessie
 
 RUN apt-get update && apt-get install -y apparmor bash-completion btrfs-tools build-essential curl ca-certificates debhelper dh-apparmor dh-systemd git libapparmor-dev libdevmapper-dev libsqlite3-dev libsystemd-journal-dev --no-install-recommends && rm -rf /var/lib/apt/lists/*
 
-ENV GO_VERSION 1.5.1
+ENV GO_VERSION 1.4.3
```
Comment (tianon) [inline contrib/builder/deb/debian-jessie/Dockerfile]: I think a rebase went horribly wrong somewhere. :smile:

**Comment by tianon** [comment]
What's `hack/make/build-old-deb` about?

**Comment by jfrazelle** [comment]
dammit

**Comment by jfrazelle** [comment]
branch mess up i was playing w something

**Comment by jfrazelle** [comment]
well shit

**Comment by jfrazelle** [comment]
i have to start over

**Comment by jfrazelle** [comment]
ok fixed wow

**Comment by tianon** [comment]
Nice, LGTM :+1:

Where in the process do the new `.deb` files actually get pushed?  Does that happen automatically on the APT server itself once the tag is up, or is that just a separate out-of-band process right now?

**Comment by jfrazelle** [comment]
its out of process now but im working on scripting and writing docs
seperately :) also im working on something cool for uploads that might make
everything easier / pretty

**Comment by tianon** [comment]
Cool, no worries; was just curious!  This looks great. :+1:

**Comment by jfrazelle** [comment]
can we merge :innocent:

**Comment by tianon** [comment]
@calavera do you want to take one more look at the updated version before merge?

**Comment by jfrazelle** [comment]
/me is just excite about removing ruby from the dockerfile

**Comment by cpuguy83** [comment]
I <3 Ruby but can't believe that fpm requires C-Extensions....

**Comment by cpuguy83** [comment]
Also, that's got to be like -400MB from the image.

**Comment by jfrazelle** [comment]
I know riteeeeeee lets mergeeeeeee hehe

**Comment by jfrazelle** [comment]
reping because i wanna get rid of ruby and make the image smaller :)

**Pull request merged**